contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -443,6 +443,10 @@ contract DualAggregator is OCR2Abstract, Ownable2StepMsgSender, AggregatorV2V3In
   /// @param secondaryRoundId the new secondary round id.
   event SecondaryRoundIdUpdated(uint32 indexed secondaryRoundId);
 
+  /// @notice indicates that a new report arrived from the primary feed and the report had already been stored .
+  /// @param primaryRoundId the new primary round id (if we're at the next block since the report it should be the same).
+  event PrimaryFeedUnlocked(uint32 indexed primaryRoundId);
```

Comment:
Would it make more sense to emit an event when the lock status changes, instead of only when it unlocks?

I was thinking we could call it: PrimaryFeedLockStatusUpdated(bool isLocked, uint32 roundId)

Reply:
The issue with emitting that way is that it wouldn't really give us new information, as cases where the new report comes in through the secondary feed are covered by the new tx + secondary event, and the way it's implemented currently this boolean is always set (for simplicity, which makes sense to me).

The addition of this event makes it so we keep it very simple on the metrics side and basically simply compare round Ids. The one case that cannot be covered by events alone are fallbacks as we have discussed previously, but that will be a separate effort to monitor.